File: Arrays/1.TwoSum.py
'''
Array: [1, 3, 7, 9, 2], target = 11
'''

def find_two_sum(arr, target):
    for p1 in range(len(arr)-1):
        des = target - arr[p1]
        for p2 in range(p1+1, len(arr)):
            if arr[p2] == des:
                return p1,p2
    return null

print(find_two_sum(arr=[1,3,7,9,2], target=11))

File: Arrays/1.TwoSum.js
/* 
Array: [1, 3, 7, 9, 2], target = 11
*/

const findTwoSum = (arr, target) => {
  for (let p1=0; p1<arr.length-1; p1++) {
    const des = target - arr[p1]
    for (let p2=p1+1; p2<arr.length; p2++) {
      if (arr[p2] === des) {
        return [p1, p2];
      }
    }
  }
  return null; 
}

console.log(findTwoSum(arr=[1,3,7,9,2], target=11))